cypress/e2e/login_activity.cy.ts

```diff
@@ -1,8 +1,6 @@
-const START_PATH = "http://localhost:3000/";
-
 describe("LoginActivity spec", () => {
   it("should render the ui", () => {
-    cy.visit(START_PATH, {
+    cy.visit("http://localhost:3000/", {
```

Comment:
is this change meant to be?

Reply:
I did it to avoid typescript warnings. It doesn't change almost anything.